src/LinSys/linsysmerger.ci

```diff
@@ -200,7 +200,7 @@ module linsysmerger {
       entry void wait4lhs() {
         when lhsbc_complete() serial "hyprelhs" { hyprelhs(); } };
       entry void wait4rhs() {
-        when rhsbc_complete() serial "hyprerhs" { hyprerhs(); } };
+        when rhsbc_complete_rhs() serial "hyprerhs" { hyprerhs(); } };
```

Comment:
So just to make sure I understand: The alterations in this PR are not problematic, but alterations in the same vein in a different portion of the code might be?

I can look into it (I suspect it may be related to quinoa taking radically different code paths based upon preprocessor flags), but I would suggest either accepting this as individually correct (as it seems to be) or changing the nature of the PR to indicate that its scope has drastically changed to an entire codebase fix rather than a quick "That looks wrong" feature branch. Either one works and is a project management choice, but it is useful for documentation purposes to understand what the purpose of a PR is.
And it would help me to know if I am tasked with fixing the entire codebase or just the portion I noticed while working on a different feature branch altogether.